test/unit_tests/model/ImageTest.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * Xournal++
+ *
+ * This file is part of the Xournal UnitTests
+ *
+ * @author Xournal++ Team
+ * https://github.com/xournalpp/xournalpp
+ *
+ * @license GNU GPLv2 or later
+ */
+
+#include <fstream>
+
+#include <config-test.h>
+#include <gtest/gtest.h>
+
+#include "model/Image.h"
+
+
+TEST(Image, testGetImageApplyOrientation) {
+    auto image = Image();
+
+    // Image width is 500px and height 130px - but has exif data saying image should be
+    // rotated 90 deg CW to have correct orientation.
+    std::ifstream imageFile{GET_TESTFILE("images/r90.jpg"), std::ios::binary};
+    auto imageData = std::string(std::istreambuf_iterator<char>(imageFile), {});
+
+    GdkPixbufLoader* loader = gdk_pixbuf_loader_new();
+    gdk_pixbuf_loader_write(loader, reinterpret_cast<const guchar*>(imageData.c_str()), imageData.length(), nullptr);
+    gdk_pixbuf_loader_close(loader, nullptr);
+
+    GdkPixbuf* pixbuf = gdk_pixbuf_loader_get_pixbuf(loader);
+
+    // Image size before  orientation
+    auto origImageSize = std::make_pair(gdk_pixbuf_get_width(pixbuf), gdk_pixbuf_get_height(pixbuf));
+    auto rotatedImageSize = std::make_pair(origImageSize.second, origImageSize.first);
```

Comment:
Please assert that the ground truth rotated image has the expected size 100x500. Since we have the exact image, we want to be absolutely sure that the ground truth matches our expectations.

Reply:
@jhilmer Could you please address @Technius comment? It would be nice to merge this PR soon.

Reply:
Done